pkg/scaffold/api.go

```diff
@@ -212,14 +220,12 @@ func (api *API) scaffoldV2() error {
 			return fmt.Errorf("error updating kustomization.yaml: %v", err)
 		}
 
-		if !api.resourceExists() {
```

Comment:
Why do you remove the check if it already exists? It is needed to prevent having duplicates in `PROJECT` if you call the same `kubebuilder create api` twice.

Reply:
It is done. The check was just centralized in the Validation func where validates all.
Closing this one a sort out.